Formula/howdoi.rb

```diff
@@ -0,0 +1,57 @@
+class Howdoi < Formula
+  include Language::Python::Virtualenv
+
+  desc "Instant coding answers via the command-line"
+  homepage "https://github.com/gleitz/howdoi/"
+  url 'https://pypi.python.org/packages/67/e0/18ca30a65aeed25c6d6fe7171d71601b3a772bf0b7b8235b9e98d3cb889e/howdoi-1.1.12.tar.gz'
```

Comment:
The audit is saying that this line should read exactly like:

`url "https://files.pythonhosted.org/packages/67/e0/18ca30a65aeed25c6d6fe7171d71601b3a772bf0b7b8235b9e98d3cb889e/howdoi-1.1.12.tar.gz"`